### DISCUSSION TITLE
Shortwave radiation: major differences between ARPEGE and other models

### DISCUSSION THREAD

**Discussion by mthiboust** [open-meteo/open-meteo#879] [General]
I am surprised by the significative difference between ARPEGE and other models for `shortwave_radiation` data. For this parameter, the ARPEGE model is consistently 20% to 50% above the other ones (ECWMF and DWD forecasts & ERA5 reanalysis). Do you know if ARPEGE and ECWMF `ssrd` GRIB parameter is measuring the same thing defined [here](https://codes.ecmwf.int/grib/param-db/169)?

This difference is so big it cannot "just" be a prediction error! I'm sure there is a rational explanation behind this but haven't found it yet. Do you have any idea?

This plot using the historical forecast data shows this well:
![image](https://github.com/open-meteo/open-meteo/assets/14574229/7a75a825-2120-4f48-9e35-006b8324bd6b)

**Comment by patrick-zippenfenig** [comment]
See here: https://github.com/open-meteo/open-meteo/issues/881

**Reply by mthiboust** [reply]
Thanks for your answer!
